src/main/java/janet/Deadline.java

```diff
@@ -0,0 +1,95 @@
+package janet;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+/**
+ * Represents a Deadline, with a description, symbol and due date.
+ */
+public class Deadline extends Task {
+    private final LocalDateTime dueDate;
+
+    Deadline(String inputLine) throws JanetException {
+        // inside the program this will be called
+        super(createDeadlineCommand(inputLine).getDescription(), createDeadlineCommand(inputLine).getSymbol());
+        this.dueDate = createDeadlineCommand(inputLine).dueDate;
+    }
+
+    Deadline(String description, String symbol, LocalDateTime dueDate) {
+        // this is used inside the static method: createDeadlineCommand
+        super(description, symbol);
+        this.dueDate = dueDate;
+    }
+
+
+    /**
+     * Returns a String array based on user's commands.
+     * First element = a String of janet.Deadline's description.
```

Comment:
I like how your JavaDoc is very informative and well-structured!

Reply:
Thanks!